.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{

```

Comment:
Do we want ide specific files in the source?
@Pasarus

Reply:
I've seen some repos have it. Can we helpful for others developing on VS Code in the future.